operators/src/source/gdal_source/loading_info.rs

```diff
@@ -412,10 +412,14 @@ impl Iterator for NetCdfCfGdalLoadingInfoPartIterator {
             return None;
         }
 
-        let steps_between = self
+        let steps_between = match self
```

Comment:
why use a match instead of a `?`?

Reply:
You cannot use ? because it is a Result that is wrapped in an Option.